docs/java-api/index.asciidoc

```diff
@@ -44,12 +44,12 @@ You need to also include Log4j 2 dependencies:
 <dependency>
     <groupId>org.apache.logging.log4j</groupId>
     <artifactId>log4j-api</artifactId>
-    <version>2.7</version>
+    <version>2.8.2</version>
```

Comment:
I meant in the doc itself that you can define it (not between the src and doc). So at least in the doc you can reuse the same variable.